CI Publish fails

Seems that the publish CI run exits with errors

```
Run npm run publish --pat ***
  npm run publish --pat ***
  shell: /usr/bin/bash -e {0}

> probe-rs-debugger@0.22.1 publish
> vsce publish --no-yarn --skip-duplicate ***

Executing prepublish script 'npm run vscode:prepublish'...

> probe-rs-debugger@0.22.1 vscode:prepublish
> rimraf dist && npm run -S build -- --minify


> probe-rs-debugger@0.22.1 build
> esbuild ./src/extension.ts --bundle --tsconfig=./tsconfig.json --external:vscode --format=cjs --platform=node --outfile=dist/extension.js --minify


  dist/extension.js  9.1kb

⚡ Done in 4ms
Error: Invalid version ***
Error: Process completed with exit code 1.
``

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -26,4 +26,6 @@ jobs:
       - run: npm install -g vsce
       - run: npm run pretest
       - name: Finally, publish to the marketplace.
-        run: npm run publish --pat ${{ secrets.VS_MARKETPLACE_TOKEN }}
+        run: npm run publish
+        env:
+          VSCE_PAT: ${{ secrets.VS_MARKETPLACE_TOKEN }}

